lametro/events.py

```diff
@@ -105,10 +107,9 @@ def _merge_events(self, events):
                 if matches:
                     spanish_audio, = matches
                     spanish_audio['Label'] = 'Audio (SAP)'
-                    event['audio'] = [english_audio, spanish_audio]
-                    continue
+                    event_audio.append(spanish_audio)
 
-            event['audio'] = [english_audio]
+            event['audio'] = event_audio
```

Comment:
I think an else would still be clearer, but this is okay.

Since you are already checking to see if english audio is available, you might think about whether you want to pass an empty list if it's not instead of having to check "not available" in two places

Reply:
Yet another great point. I think the empty list makes the logic (without any `else` clauses) easier to grok, too.